spec/controllers/api/did_not_votes_controller_spec.rb

```diff
@@ -0,0 +1,28 @@
+require 'rails_helper'
+
+describe API::DidNotVotesController do
+
+  let(:user)                  { create :user }
+  let(:group)                 { create :group }
+  let(:discussion)            { create :discussion, group: group }
+  let(:motion)                { create :motion, discussion: discussion }
+
+  before do
+    motion
+    stub_request(:post, "http://localhost:9292/faye").to_return(status: 200)
+    group.admins << user
+    sign_in user
+    MotionService.close(motion)
+  end
+
+  describe 'index' do
+
+    context 'success' do
+      it 'returns did not votes for a motion' do
+        get :index, motion_id: motion.id, format: :json
+        json = JSON.parse(response.body)
+        expect(json.keys).to include *(%w[did_not_votes])
```

Comment:
I'd love to see a check for permissions in this test, the way we do with most of the other index actions

```
group_member = create :user
not_group_member = create :user
groups.members << group_member
...
expect(user_ids).to include group_member.id
expect(user_ids).to_not include not_group_member.id
```
^^ (The gist, anyway)